backend/src/controllers/job.controller.ts

```
@@ -38,29 +39,41 @@ export class JobsController extends Controller {
     });
-    const jobsTypeList: JobType[] = ['completed', 'failed', 'active', 'delayed', 'waiting', 'waiting-children', 'paused', 'repeat', 'wait']
+    const jobsTypeList: JobType[] = ['completed', 'failed', 'active', 'delayed', 'waiting', 'waiting-children', 'paused', 'repeat', 'wait', 'prioritized']
     if (jobsTypeList.includes(jobsType)) {
       const jobs = await jobQueue.getJobs(jobsType);
-      jobs.forEach((j: any) => delete j.data.randomKey)
-      return { jobs };
+      let jobsUser;
+      if (scopes.includes('admin')) {
+        jobsUser = jobs;
+        // admin should not see randomKey
+        jobsUser.forEach((j: any) => j.data.randomKey = undefined)
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)